src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -916,24 +917,24 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Condition="'$(TargetsWindows)' == 'true' and '$(IsUAPAssembly)' != 'true'" Include="Microsoft.Win32.Registry" Version="$(MicrosoftWin32RegistryVersion)" />
+    <PackageReference Condition="'$(TargetsWindows)' == 'true' and '$(IsUAPAssembly)' != 'true' and ('$(TargetGroup)' == 'netstandard' or ('$(TargetGroup)' == 'netcoreapp' and '$(_TargetFrameworkVersionWithoutV)' &lt; '6.0'))" Include="Microsoft.Win32.Registry" Version="$(MicrosoftWin32RegistryVersion)" />
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="$(SystemConfigurationConfigurationManagerVersion)" />
     <PackageReference Include="System.Security.Permissions" Version="$(SystemSecurityPermissionsVersion)" />
```

Comment:
This can be removed as System.Configuration.ConfigurationManager references it, and System.Configuration.ConfigurationManager can be removed as System.Runtime.Caching references it (at least for .NET 6+ for now).